include/nccl_ofi_param.h

```diff
@@ -118,19 +151,21 @@ static inline const char *ofi_nccl_##name() { \
     return value; \
 }
 
+#endif
+
 /*
  * Enable using endpoints with IPv6 addressing format for TCP provider.
  * By default, we disable using endpoints having IPv6 addressing format.
  */
-OFI_NCCL_PARAM_INT(use_ipv6_tcp, "USE_IPV6_TCP", 0);
+OFI_NCCL_PARAM_INT(use_ipv6_tcp, "USE_IPV6_TCP", 0)
```

Comment:
Can we avoid touching these? I don't think it's strictly necessary to drop the semi

Reply:
Done.  Of course, we weren't consistent with the use of semicolons, but adding the one was a smaller diff than removing all the others.